### PR TITLE
Kickstart service test: enable / disable.

### PR DESCRIPTION
**Pull request by KosiehBarter**
Tested services: sshd (disable), systemd-timesyncd (enable),
systemd-networkd (disabled in config, test).

Signed-off-by: Karel Valek <kvalek@redhat.com>

### CODE REVIEW

**Comment by KosiehBarter** [comment]
Added SELinux test to services test (see #193).

**Comment by clumens** [comment]
Ignore the "some checks have failed" message - all PRs are getting that for the moment due to anaconda itself failing in jenkins.

File: tests/kickstart_tests/services.ks [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+systemctl is-enabled sshd | grep disabled
+if [[ $? -ne 0 ]]; then
+    echo "*** sshd is enabled, not disabled" >> /root/RESULT
+fi
+
+# Test disabled - W/out change
+systemctl is-enabled systemd-networkd | grep disabled
+if [[ $? -ne 0 ]]; then
+    echo "*** systemd-networkd is enabled, should be disabled" >> /root/RESULT
+fi
```
Comment (vpodzime) [inline tests/kickstart_tests/services.ks]: You could drop the ``|grep ...`` parts as ``systemctl is-enabled`` exits with the code 0/1 if the service is enabled/disabled.

File: tests/kickstart_tests/services.ks [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+# SELinux test
+cat /etc/selinux/config | grep SELINUX=enforcing
```
Comment (vpodzime) [inline tests/kickstart_tests/services.ks]: ``grep`` takes an optional second argument that is a file that should be "grepped", so you can drop the ``cat`` part here.

**Comment by KosiehBarter** [comment]
Cherry-picked locally and pushed.

**Pull request closed without merging**